### PR TITLE
Implementing Platform Email designs.

### PR DESCRIPTION
**Pull request by BenjaminSsempala**
**_WHAT DOES THIS PR DO?_**
- [x] Platform verification email.

**_WHAT ISSUES ARE RELATED TO THIS PR?_**

- Jira cards
    - [AN-377]()
- GitHub issues
    - Closes #<enter GitHub issue number here>

**_HOW DO I TEST OUT THIS PR?_**

**_WHICH ENDPOINTS SHOULD BE READY FOR TESTING?:_** 

**_IS THERE ANY JENKINS CONSOLE LINK FOR CODE COVERAGE AND BUILD INFO?_**

**_ARE THERE ANY RELATED PRs?_**

[AN-377]: https://airqoteam.atlassian.net/browse/AN-377?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/auth-service/utils/mailer.js [hunk 1]
```diff
@@ -215,6 +215,37 @@ const mailer = {
         subject: "Verify your AirQo Platform account",
         html: msgTemplates.v2_emailVerification(firstName, user_id, token),
         bcc,
+       attachments: [
```
Comment (BenjaminSsempala) [inline src/auth-service/utils/mailer.js:218]: Adding Images and logos to the email

File: src/auth-service/utils/mailer.js [hunk 2]
```diff
@@ -215,6 +215,37 @@ const mailer = {
...
+        path: "../config/images/Twitter.png",
+        cid: "Twitter",
+        contentDisposition: "inline",
+      },
+     {
+        filename: "linkedInLogo.png",
+        path: "../config/images/linkedInLogo.png",
+        cid: "LinkedInLogo",
+        contentDisposition: "inline",
+      }],
```
Comment (BenjaminSsempala) [inline src/auth-service/utils/mailer.js:248]: Adding images and  logos to the email.

File: src/firebase/auth/functions/config/emailTemplates.js [hunk 3]
```diff
@@ -26,6 +26,40 @@ module.exports = {
...
+            <div style="text-align: center;">
+                <p style="">© 2022 AirQo. </p>
+
+                <p>Makerere University, Software Systems Centre, Block B, Level 3, College of Computing and Information
+                    Sciences, Plot 56 University Pool Road</p>
+            </div>
+
+        </div>
+        `
+    ;
```
Comment (BenjaminSsempala) [inline src/firebase/auth/functions/config/emailTemplates.js:63]: Adding footer to mobile-app welcome message

**Review by NoahNsimbe — APPROVED** [review]
LGTM

**Review by Baalmart — APPROVED** [review]
Thanks @BenjaminSsempala

**Pull request merged**